lib/diplomat/kv.rb

```diff
@@ -112,6 +114,7 @@ def put(key, value, options = nil)
         url += check_acl_token
         url += use_cas(@options)
         url += dc(@options)
+        url += acquire(@options)
```

Comment:
I have added the other doc string. And yes I see the functionality I needed originally in the Lock.acquire method. I was following the Consul leader election guide which led me down the acquire path on the kv put.

I do see some benefit as to adding this in terms of being able to delve  intop session info for existing keys that have existing sessions.